lib/dataSource-core.js

```diff
@@ -354,8 +363,10 @@ kettle.dataSource.CouchDB.read = function (that, resp) {
  * `that.options.rules.writePayload`. By default, the entirety of the model is wrapped in a `value` element to avoid
  * collisions with top-level CouchDB variables such as `_id` and `_rev`.
  *
- * @param {model} The data to be stored.
- * @param {options} The `dataSource` options (see above).
+ * @param {Component} that - The dataSource component, used to read the payload write transform option
```

Comment:
This function is a good example of two competing styles, one which favours ending a sentence with a period, one of which lacks the trailing period.  You could argue for being consistent between docs in the same package,  but IMO you should at least be consistent within a single block.

Reply:
I've now tried to revert to a style where short, "noun-phrase" descriptions as in these params have no period, but longer or multiple sentences here or in the headline description retain them.

Reply:
That seems fair to me.